src/libs/globals.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash/fp'
+import { forwardRef } from 'react'
+import { h } from 'react-hyperscript-helpers'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+
+
+const globals = Utils.atom(JSON.parse(sessionStorage['globals'] || '{}'))
+globals.subscribe(v => {
+  if (!v) {
+    sessionStorage.removeItem('globals')
+  } else {
+    sessionStorage['globals'] = JSON.stringify(v)
+  }
+})
+
+const get = key => globals.get()[key]
+const set = _.curry((key, value) => globals.update(m => ({ ...m, [key]: value })))
+
+export const globalObserver = key => WrappedComponent => {
```

Comment:
This looks an awful lot like `connectAtom`. Perhaps we could export the atom and use that instead?